console/src/main/scripts/plugins/metrics/less/metrics.less

```diff
@@ -2458,6 +2469,7 @@ section {
 
   padding-bottom: @grid-gutter-width/2;
   padding-top: @grid-gutter-width/2;
+  height: 120px;
```

Comment:
Actually it applies to .hk-dashboard-block {}
@karelhala, did you test in different resolutions (from 768px width above)? We usually don't define heights like this since the content adjust. For this situation we should use a plugin like https://github.com/liabru/jquery-match-height (which I used in my prototype), which makes the elements of the same line with the same height.